packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -88,7 +88,9 @@ const MessageContentAsRichTextHTML = (props: ChatMessageContentProps): JSX.Eleme
   return (
     <MessageContentWithLiveAria
       message={props.message}
-      liveMessage={`${props.message.mine ? '' : liveAuthor} ${extractContent(props.message.content || '')}`}
+      liveMessage={`${props.message.editedOn ? props.strings.editedTag : ''} ${
```

Comment:
It will be a good idea to move this logic to a separate func and reuse it in both places

Reply:
Yeah, good idea. Updated